contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -99,6 +99,27 @@ contract VestingRegistryLogic is VestingRegistryStorage {
 		}
 	}
 
+	/**
+	 * @notice adds four year vestings to vesting registry logic
+	 * @param _tokenOwners array of token owners
+	 * @param _vestingAddresses array of vesting addresses
+	 */
+	function addFourYearVestings(address[] calldata _tokenOwners, address[] calldata _vestingAddresses) external onlyAuthorized {
+		require(_tokenOwners.length == _vestingAddresses.length, "arrays mismatch");
+		uint256 vestingCreationType = 4;
+		uint256 cliff = 4 weeks;
+		uint256 duration = 156 weeks;
```

Comment:
[VR4](https://sovryn.monday.com/docs/2544671802?blockId=6fcf4467-6721-403f-a0b4-3379fdb1df37) - vars can be constants